<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regestration</class>
 <widget class="QWidget" name="Regestration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 108, 158);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);

border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);
background-color:rgb(71, 147, 175);
border-radius:8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);

border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);
background-color:rgb(71, 147, 175);
border-radius:8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);

border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PswdEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);
background-color:rgb(71, 147, 175);
border-radius:8px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PswdEdit2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);
background-color:rgb(71, 147, 175);
border-radius:8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);

border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Repeat password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);

border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SiginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial'; /* Встановити сімейство шрифтів для вкладок */
  font-size: 14px; /* Встановити розмір шрифту для вкладок */
  font-weight: bold; 
color:rgb(255, 196, 112);
background-color:rgb(71, 147, 175);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Sigin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
